lib/plugins/basalprofile.js

```diff
@@ -156,6 +156,20 @@ function init (ctx) {
     }]
   };
 
+  function googleHomeCurrentBasalhandler (result, next, sbx) {
+    var pwd = result.parameters && result.parameters.givenName ? result.parameters.givenName : null;
+    next(basalMessage(pwd, sbx));
+  }
+
+  basal.googleHome = {
+    intentHandlers: [{
+      intent: 'CurrentMetric'
+      , routableSlot:'metric'
+      , slots:['basal', 'current basal']
+      , intentHandler: googleHomeCurrentBasalhandler
+    }]
+  };
+  
```

Comment:
What API call changes are needed to have this live in the Home plugin? basalprofile.js should only contain basalprofile code.